pkg/agent/as3/as3Common.go

```diff
@@ -60,6 +60,8 @@ func (am *AS3Manager) validateAS3Template(template string) bool {
 }
 
 func (am *AS3Manager) processResourcesForAS3(sharedApp as3Application) {
+	defer log.Timeit("info")("")
```

Comment:
Please change info to debug as at info level we shouldn't log function execution time.

Reply:
done